selfdrive/car/mazda/carstate.py

```diff
@@ -168,6 +219,55 @@ def get_can_parser(CP):
         ("BSM", 10),
       ]
 
+    if CP.carFingerprint in GEN2:
+      signals += [
+        ("CRZ_SPEED", "CRUZE_STATE", 0),
+        ("CRZ_ENABLED", "CRUZE_STATE", 0),
+        ("PRE_ENABLE", "CRUZE_STATE", 0),
+        ("BRAKE_PEDAL_PRESSED", "BRAKE_PEDAL", 0),
+        ("SET_P", "CRZ_BTNS",0),
+        ("SET_M", "CRZ_BTNS",0),
+        ("RES", "CRZ_BTNS",0),
+        ("CAN", "CRZ_BTNS",0),
+        ("MODE_ACC", "CRZ_BTNS",0),
+        ("MODE_LKAS", "CRZ_BTNS",0),
+        ("DISTANCE_P", "CRZ_BTNS",0),
+        ("DISTANCE_M", "CRZ_BTNS",0),
+        ("STATIC_0", "CRZ_BTNS",0),
+        ("STATIC_1", "CRZ_BTNS",0),
+        ("STATIC_2", "CRZ_BTNS",0),
+        ("CTR", "CRZ_BTNS", 0),
+        ("LEFT_BLINK", "BLINK_INFO", 0),
+        ("RIGHT_BLINK", "BLINK_INFO", 0),
+        ("ACCEL_CMD", "ACC", 0),
+        ("HOLD", "ACC", 0),
+        ("RESUME", "ACC", 0),
+        ("NEW_SIGNAL_1", "ACC", 0),
```

Comment:
Can we understand what these signals are so we can set them appropriately? Or is there a reason to pass them through?

Reply:
I haven't figured out what they do. All I know is that I have seen them change.